ChangeLog.md

```diff
@@ -1,3 +1,12 @@
+2015/05/13 Version 0.8.2
+- Updated dev depedencies and fixed all failing tests
+
+2015/05/12 Version 0.8.1
+- Fixed an issue where .types file is in another directory that the root directory
+
+2015/05/11 Version 0.8.0
```

Comment:
I should probably add something about the changelog in the contributing guidelines---I like to manage this myself while bumping and publishing releases. But the notes here are really helpful! I'll just end up munging this a bit.